pkg/maps/ipcache/ipcache.go

```diff
@@ -118,19 +118,45 @@ func NewKey(ip net.IP, mask net.IPMask, clusterID uint16) Key {
 	return result
 }
 
+// RemoteEndpointInfoFlags represents various flags that can be attached to
+// remote endpoints in the IPCache.
+type RemoteEndpointInfoFlags uint8
+
+// String returns a human-readable representation of the flags present in the
+// RemoteEndpointInfoFlags.
+// The output format is the string name of each flag contained in the flag set,
+// separated by a comma. If no flags are set, then "<none>" is returned.
+func (f RemoteEndpointInfoFlags) String() string {
+	// If more flags are added in the future, then this method will need
+	// a re-work to support multiple flags.
+	// Right now, it only supports checking for FlagSkipTunnel.
+	if f&FlagSkipTunnel == FlagSkipTunnel {
```

Comment:
is this how Go formats it? I mean `x&y` vs. `x & y` (spaces around `&`)

Reply:
Yes, something about highlighting operator precedence.